node/fixup_pj_files_for_build_type

```diff
@@ -0,0 +1,15 @@
+cat >build-ts/cjs/package.json <<!EOF
```

Comment:
Can you add a comment inside the file with description of what it does / what is it used for?